### DISCUSSION TITLE
Why can't we use DbBlockIdGenerator on Sqlite?

### DISCUSSION THREAD

**Discussion by hyzx86** [OrchardCMS/OrchardCore#14662] [General]
I've checked the code, why don't we also use DbBlockIdGenerator in Sqlite?

I would love to be able to reuse this functionality, such as adding autoid to a specified function module or field, but DefaultIdGenerator needs to qualify specific tables

https://github.com/sebastienros/yessql/blob/main/src/YesSql.Core/Services/DbBlockIdGenerator.cs

https://github.com/OrchardCMS/OrchardCore/blob/5a472d483fc360d1e1aa69dc7e098be6a038e62c/src/OrchardCore/OrchardCore.Data.YesSql/OrchardCoreBuilderExtensions.cs#L69-L98

**Comment by sebastienros** [comment]
Have you checked in YesSQL if there is a comment explaining the reason?

**Reply by hyzx86** [reply]
I didn't find any instructions for that 😢
